### PR TITLE
Update config.lua (french)

### PR DESCRIPTION
**Pull request by Hantunaar**
Update for better fitting to french language.

### CODE REVIEW

File: Locale/frFR/config.lua [hunk 1]
```diff
@@ -2,180 +2,180 @@ QuestieLocale.locale['frFR'] = {
...
+	['ENABLE_QUESTIE_DESC'] = "Active les icônes de Questie",
+	['ENABLE_ICON'] = "Activer le bouton autour de la mini-carte",
+	['ENABLE_ICON_DESC'] = "Active le bouton Questie autour de la mini-carte. Le panneau de configuration reste accessible avec la commande /questie.",
+	['ENABLE_INSTANT'] = "Activer Texte instantané (quêtes)",
+	['ENABLE_INSTANT_DESC'] = "Active l'option Texte instantané (quêtes). Cela revient à cocher l'option du même nom dans Interface/Affichage.",
+	['LEVEL_HEADER'] = "Options du niveau des quêtes",
+	['ENABLE_LOWLEVEL'] = "Afficher les quêtes obsolètes",
+	['ENABLE_LOWLEVEL_DESC'] = "Active l'affichage des quêtes de bas niveaux sur la carte.",
+	['LOWLEVEL_BELOW'] = "<Niveaux inférieurs max>",
+	['LOWLEVEL_BELOW_DESC'] = "Définis l'écart maximal entre le niveau des quêtes et celui du joueur. (Par défaut : %s)",
```
Comment (BreakBB) [inline Locale/frFR/config.lua]: I am not a fan of a whitespace before a colon. Could you change this and the below occurrences?

```suggestion
	['LOWLEVEL_BELOW_DESC'] = "Définis l'écart maximal entre le niveau des quêtes et celui du joueur. (Par défaut: %s)",
```

File: Locale/frFR/config.lua [hunk 2]
```diff
@@ -2,180 +2,180 @@ QuestieLocale.locale['frFR'] = {
...
-	['ICON_LEFT_CLICK'] = "Clic Gauche",
-	['ICON_TOGGLE'] = "Options de bascule",
-	['ICON_SHIFTLEFT_CLICK'] = "Maj + Clic Gauche",
-	['ICON_TOGGLE_QUESTIE'] = "Basculer la Questie",
-	['ICON_CTRLRIGHT_CLICK'] = "Ctrl + Clic Droit",
-	['ICON_CTRLLEFT_CLICK'] = "Ctrl + Clic Gauche",
-	['ICON_HIDE'] = "Bouton Masquer la mini-carte",
-	['ICON_RIGHT_CLICK'] = "Clic Droit",
-	['ICON_JOURNEY'] = "Basculer mon voyage",
+	['ICON_LEFT_CLICK'] = "Clic Gauche ",
```
Comment (BreakBB) [inline Locale/frFR/config.lua]:
```suggestion
	['ICON_LEFT_CLICK'] = "Clic Gauche",
```

File: Locale/frFR/config.lua [hunk 3]
```diff
@@ -2,180 +2,180 @@ QuestieLocale.locale['frFR'] = {
...
+	['ICON_TOGGLE'] = "Panneau de configuration",
+	['ICON_SHIFTLEFT_CLICK'] = "Maj + Clic Gauche ",
```
Comment (BreakBB) [inline Locale/frFR/config.lua]:
```suggestion
	['ICON_SHIFTLEFT_CLICK'] = "Maj + Clic Gauche",
```

File: Locale/frFR/config.lua [hunk 4]
```diff
@@ -2,180 +2,180 @@ QuestieLocale.locale['frFR'] = {
...
+	['ICON_TOGGLE_QUESTIE'] = "Afficher/cacher Questie",
+	['ICON_CTRLRIGHT_CLICK'] = "Ctrl + Clic Droit ",
```
Comment (BreakBB) [inline Locale/frFR/config.lua]:
```suggestion
	['ICON_CTRLRIGHT_CLICK'] = "Ctrl + Clic Droit",
```

File: Locale/frFR/config.lua [hunk 5]
```diff
@@ -2,180 +2,180 @@ QuestieLocale.locale['frFR'] = {
...
+	['ICON_CTRLLEFT_CLICK'] = "Ctrl + Clic Gauche ",
```
Comment (BreakBB) [inline Locale/frFR/config.lua]:
```suggestion
	['ICON_CTRLLEFT_CLICK'] = "Ctrl + Clic Gauche",
```

File: Locale/frFR/config.lua [hunk 6]
```diff
@@ -2,180 +2,180 @@ QuestieLocale.locale['frFR'] = {
...
+	['ICON_HIDE'] = "Cacher le bouton autour de la mini-carte",
+	['ICON_RIGHT_CLICK'] = "Clic Droit ",
```
Comment (BreakBB) [inline Locale/frFR/config.lua]:
```suggestion
	['ICON_RIGHT_CLICK'] = "Clic Droit",
```

**Comment by BreakBB** [comment]
Thanks for these PR @Hantunaar. I've added some comments and it would be great if you could look over them!

File: Locale/frFR/config.lua [hunk 7]
```diff
@@ -216,82 +216,82 @@ QuestieLocale.locale['frFR'] = {
...
+	['JOURNEY_QUEST_LEVEL'] = "Niveau recommandé :",
+	['JOURNEY_QUEST_MINLEVEL'] = "Niveau minimum requis pour la quête :",
+	['JOURNEY_QUEST_ID'] = "ID de quête :",
+	['JOURNEY_DIFFICULTY'] = "Fourchette de difficulté : %s",
 	['JOURNEY_AUTO_QUEST'] = "Cette quête est une quête d'achèvement automatique et ne contient pas d'objectif.",
 	['JOURNEY_RECENT_EVENTS'] = "Votre historique récent",
-	['JOURNEY_NOTE_CREATED'] = "Vous avez créé une nouvelle note intitulée: %s",
-	['JOURNEY_LEVELUP'] = "Félicitations! Vous avez atteint %s!",
+	['JOURNEY_NOTE_CREATED'] = "Vous avez créé une nouvelle note intitulée : %s",
+	['JOURNEY_LEVELUP'] = "Félicitations ! Vous avez atteint %s !",
```
[the remark this answers is not in the document]
Comment (Hantunaar) [reply Locale/frFR/config.lua]: Yes, there should always be a whitespace before and after ? ! : and ; in french. Dots follow the same rule as in english though. =)

[the remark this answers is not in the document]
Comment (BreakBB) [reply Locale/frFR/config.lua]: Ah I see. Well then ignore my comments on those! Good to know :)

File: Locale/frFR/config.lua (hunk 6 above)
Comment (Hantunaar) [reply Locale/frFR/config.lua]: I added whitespaces in these ones so it follow the french whitespace rule for the " : " sign. This way the whole sentence looks like "Clic Gauche : Panneau de configuration" with "Clic Gauche" being displayed in white/grey and " : Panneau de configuration" in yellow.

**Pull request merged**